api/project.go

```diff
@@ -299,7 +306,19 @@ func GetBaseDatabaseName(databaseName, dbNameTemplate, labelsJSON string) (strin
 	return names[1], nil
 }
 
-func getTemplateTokens(template string) []string {
+func getTemplateTokens(template string) ([]string, []string) {
 	r := regexp.MustCompile(`{{[^{}]+}}`)
-	return r.FindAllString(template, -1)
+	tokens := r.FindAllString(template, -1)
+	if len(tokens) > 0 {
+		split := r.Split(template, -1)
+		fixed := make([]string, 0, len(split)-len(tokens))
```

Comment:
nit:

```
var fixed []string
```

Reply:
Pre-allocation is used here and performance is a little better.

Reply:
For a database core engine, performance is important but I think Bytebase aims for simplicity than performance. We use "var x []string" in other places, so let's keep them consistent.

Reply:
got it.